Frontend/speedcart-react/src/pages/Dashboard/Dashboard.tsx

```diff
@@ -4,12 +4,12 @@ import { Link } from 'react-router-dom';
 import { FaTrash, FaEdit, FaShare, FaClipboard } from 'react-icons/fa';
 import { useAuth, fetchOwnedShoppingLists, fetchSharedShoppingLists, deleteShoppingList, createShareLink, AuthContextType } from 'shared';
 import Modal from '@components/Modal';
-import StatusModal from '@components/StatusModal';
 import CustomCheckbox from '@components/CustomCheckbox';
 
 import { RequestStatus } from '@constants/enums';
 
 import styles from './Dashboard.module.css';
+import PageLayout from '@components/PageLayout';
```

Comment:
Move import up with other `@component` imports, specifically make it the first one among them.